#rectangle tag
string label
#rectangle width and height
uint32 width
uint32 height
#rectangle top X and Y coordinates
uint32 topX
uint32 topY
#rectangle bottom X and Y coordinates 
#so that we can display it in OpenCV
uint32 bottomX
uint32 bottomY
